code/socialDistribution/models.py

```diff
@@ -127,4 +128,50 @@ def is_public(self):
         '''
         Check if post is public
         '''
-        return self.visibility == self.PostVisibility.PUBLIC
\ No newline at end of file
+        return self.visibility == self.PostVisibility.PUBLIC
+
+    def when(self):
```

Comment:
Looks good but what do you think about using a library for this?
https://pypi.org/project/timeago/